src/components/Layout/Layout.js

```diff
@@ -24,6 +25,7 @@ const Layout = ({
 
   return (
     <div className={styles.layout}>
+      <Navbar/>
```

Comment:
Shoud be replaced with `<Navbar />`. Spacing is important. Interesting that is passed the linter.